src/aws-adapter.js

```diff
@@ -152,6 +152,13 @@ async function lambda(evt, ctx) {
       evt.rawQueryString = searchParams.toString();
       evt.headers = {};
       evt.requestContext = { http: {} };
+
+      // function was invoked by a trigger
+      if (!evt.body && evt.Records && evt.Records.length === 1) {
+        evt.requestContext.http.method = 'POST';
+        evt.body = evt.Records[0].body;
+        evt.headers['content-type'] = 'application/json';
+      }
```

Comment:
Deliberately only done for 1 record at a time, so the function will behave as if invoked manually.

Reply:
why not pass in all the records ? there might me actions that can handle several message at once (eg the forms handler)